iTwin/iTwinUI
# fix(Table): Date filter checks

## Changes

The date range picker used in date filters now prevents users from picking a start date later than the end date or picking an end date earlier than the start date. When selecting a `From` date, any dates after a given `To` date are disabled. When selecting a `To` date, any dates before a given `From` date are disabled. 

## Testing

1. Open the `Filters` story in the Table storybook
2. Click the date filter button from the `Start date` or `End date` columns
3. Set the `From` date
4. Open the `DatePicker` for `To` date and notice how dates before the `From` date are disabled
5. Set the `To` date
6. Open the `DatePicker` for `From` date and notice how dates after the `To` date are disabled

## Docs

Changeset added
